.github/issue_template.md

```diff
@@ -1,21 +1,21 @@
-<!-- Thanks for the interest in Spring Cloud Data Flow! Before you create a new issue, please answer the following question.-->
+<!-- Thanks for your interest in Spring Cloud Data Flow! Before you create a new issue, please answer the following question.-->
 
-**Is this a "Bug Report" or "Feature Request"?** (choose one and remove the irrelevant sections before submitting the issue):
+**Is this a "Bug Report" or "Feature Request"?** (choose one and remove the irrelevant sections before you submit the issue):
 
 <!--
-If this is a "Bug Report", please:
+If this is a "Bug Report", please provide the following information:
 
-Describe:
-A clear and concise description of what the bug is. Additionally, it'd help if you can include the logs and entire stacktrace including the "caused by". (see GitHub-Markdown docs at: https://guides.github.com/features/mastering-markdown for logs/code formatting guidelines)_
+Description:
+A clear and concise description of what the bug is. Additionally, it would help if you could include the logs and the entire stacktrace, including the "caused by" portion. (see GitHub-Markdown docs at: https://guides.github.com/features/mastering-markdown for logs/code formatting guidelines)_
 
 Release versions:
-There's an API (http://docs.spring.io/spring-cloud-dataflow/docs/current/reference/htmlsingle/#api-guide-resources-server-meta-retrieving) to gather SCDF's system information including the dependent projects and the associated versions. Alternatively, you can capture this information from the Dashboard's About tab (http://docs.spring.io/spring-cloud-dataflow/docs/current/reference/htmlsingle/#dashboard), so make sure to include the copied JSON in the bug report.
+There is an API (http://docs.spring.io/spring-cloud-dataflow/docs/current/reference/htmlsingle/#api-guide-resources-server-meta-retrieving) to gather SCDF's system information, including the dependent projects and the associated versions. Alternatively, you can capture this information from the Dashboard's About tab (http://docs.spring.io/spring-cloud-dataflow/docs/current/reference/htmlsingle/#dashboard). Please be sure to include the copied JSON in the bug report.
 
-Custom Apps:
-If your Stream or Task data pipeline includes custom apps and there's a problem associated with it, it'd be good to review the sample-app (add a link to GitHub repo) and the release versions in use. Make sure to also share the register, create, and deploy/launch DSL commands for completeness.
+Custom apps:
+If your Stream or Task data pipeline includes custom apps and there is a problem associated with it, please review the sample-app (add a link to the GitHub repo) and the release versions in use. Also, please be sure to share the register, create, and deploy/launch DSL commands for completeness.
```

Comment:
> please review the sample-app

_perhaps_

> please share the sample-app

(yes, we want to review their app/code)

Reply:
👍